.github/workflows/test.yml

```diff
@@ -4,14 +4,14 @@ on:
     branches:
       - main
     paths-ignore:
-      - 'docs/**'
-      - '*.md'
+      - "docs/**"
+      - "*.md"
```

Comment:
I think we need to remove this as prettier formats the markdown document, no?

Reply:
But pre-commit runs separately (on pre-commit.ci), it's not part of this job?

Reply:
(Same issue would be true with end-of-file checks, etc)